plugins/services/src/js/utils/CreateServiceModalFormUtil.js

```diff
@@ -38,6 +40,72 @@ const CreateServiceModalFormUtil = {
 
       return memo;
     }, baseObject);
+  },
+
+  /**
+   * Maps errors to the tabs they're coming from
+   *
+   * @param {Array} errors - an array of all errors
+   * @param {Object} tabPathRegexes - an object where keys map to the service form tabs,
+   * and the values are an array of regexes to test error paths against
+   * @param {Array} pathTranslationRules - The path translation rules
+   * @param {object} i18n - lingui internationalization instance object
+   * @returns {Object} An object where where keys map to the service form tabs,
+   * and the values are an array of objects containing error data
+   */
+  getTopLevelTabErrors(errors, tabPathRegexes, pathTranslationRules, i18n) {
+    const errorPaths = errors.map(error => error.path.join("."));
+
+    return errorPaths.reduce((errorsByTabObj, pathString) => {
+      Object.keys(tabPathRegexes).forEach(tabId => {
```

Comment:
Is it possible for this to not have keys?

Reply:
I saw it's a constants file. Why do we not use it directly here?

Reply:
I was originally using it directly, but I changed it to a parameter when I was writing tests. If we do something like this on other forms, we can move this to another util file and pass a different object of the same shape.

If you or any other reviewers feel strongly that it should be used directly, I think it's ok to change.